packages/checkout/widgets-lib/src/locales/en.json

```diff
@@ -231,30 +231,36 @@
     },
     "TOP_UP_VIEW": {
       "header": {
-        "title": "How would you like to add coins?"
+        "title": "How would you like to add tokens?"
```

Comment:
Must remain as `coins` unless updating this across all relevant screens

Reply:
The designs shared by UI/UX said tokens in there.